[libpas] Add libpas.xcodeproj missing schemes

zakariaridouh opened the pull request:
#### 5edb8d24ba49bbaac771fb5566c4817a40c48135
<pre>
[libpas] Add libpas.xcodeproj missing schemes
<a href="https://bugs.webkit.org/show_bug.cgi?id=285702">https://bugs.webkit.org/show_bug.cgi?id=285702</a>
<a href="https://rdar.apple.com/142635464">rdar://142635464</a>

Reviewed by Elliott Williams.

There were multiple schemes  missing from the `libpas.xcodeproj` file, such as `test_pas`.
Added them back in this patch.

* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/all.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/mbmalloc_hotbit.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/mbmalloc_iso_common_primitive.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/pas.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/test_pas.xcscheme: Added.

Canonical link: <a href="https://commits.webkit.org/288784@main">https://commits.webkit.org/288784@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/126fda6eb8b7dc03eea01f34c01cc9906d6b2199

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/84304 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/3927 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/38609 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/89382 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/35313 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/4013 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/11902 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/65566 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/23404 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/87350 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/3012 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/76600 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/45859 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/2963 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/30830 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/34362 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/77268 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/73860 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/31598 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/90763 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/83321 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/11571 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/8427 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/74020 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/11798 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/72425 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/73220 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/18136 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/17543 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/15987 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/2936 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/11523 "Built successfully") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/105740 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/11372 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/25527 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/14848 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/13145 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

emw-apple reviewed (approved):
naming nit: These files you are adding are schemes, not targets. Could you reword the commit message to say that you're adding schemes?

Since these schemes are all in libpas.xcodeproj, they won't bubble up to the WebKit workspace and pollute everyone's scheme menu. I'd probably prefer having only one or two schemes that build multiple targets at once. But this project is libpas's thing, so I have no strong opinion about how it's set up.

rs=me, cc @stwrt

zakariaridouh committed:
[libpas] Add libpas.xcodeproj missing schemes
https://bugs.webkit.org/show_bug.cgi?id=285702
rdar://142635464

Reviewed by Elliott Williams.

There were multiple schemes  missing from the `libpas.xcodeproj` file, such as `test_pas`.
Added them back in this patch.

* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/all.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/mbmalloc_hotbit.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/mbmalloc_iso_common_primitive.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/pas.xcscheme: Added.
* Source/bmalloc/libpas/libpas.xcodeproj/xcshareddata/xcschemes/test_pas.xcscheme: Added.

Canonical link: https://commits.webkit.org/288784@main
